Linux_scripts/RHUI_repo_validation_scripts/repo_check.py

```diff
@@ -87,6 +107,8 @@ def yum_repo_check():
          my_timer.start()
          stdout, stderr = yum.communicate()
          return stderr
+      except:
+         logging.error("There seems to be some issue while processing yum check-update command output, rerun the script after fixing the issue")
```

Comment:
what is the customer supposed to do?

Reply:
There might be some chances like yum check-update is hung due to performance issues or any other config issues, CX need to manually execute this command to check whats happening.

Reply:
please instruct the customer to manually run the given command here